bindings/java/src/org/sleuthkit/datamodel/BlackboardAttribute.java

```diff
@@ -518,8 +521,32 @@ public String getDisplayString() {
 				return Double.toString(getValueDouble());
 			case BYTE:
 				return bytesToHexString(getValueBytes());
-			case DATETIME:
-				return FsContent.epochToTime(getValueLong());
+				
+			case DATETIME: {
+				
+				// return the date/time string in the timezone associated with the datasource,
+				TimeZone tzone = TimeZone.getDefault();
+				
+				try {
+					final Content dataSource = getParentArtifact().getDataSource();
+					if ((dataSource != null) && (dataSource instanceof Image )) {
+						Image  image = (Image) dataSource;
+						tzone = TimeZone.getTimeZone(image.getTimeZone());
+					} 
+				} catch (TskException ex) {
+					// contnue with TimeZone.getDefault();
+				}
+				
+				String timeString = "0000-00-00 00:00:00";
+				long epochSeconds = getValueLong();
+				if (epochSeconds != 0) {
+					synchronized (DATE_FORMATTER) {
+						DATE_FORMATTER.setTimeZone(tzone);
```

Comment:
I'd rather that this go into a new AbstractFile.epochToTime method that takes in a TimeZone argument.   The code in the attribute tries to find the Timezone and then calls either the new epochToTime() with that TimeZone or calls the original one. 

Also, epochToTime() is in AbstractFile, but all of the references are to FsContent. We should update those to make that more obvious. It took me a bit to find it.